website/source/docs/provisioners/shell-local.html.md

```diff
@@ -108,6 +108,14 @@ Optional parameters:
     **Important:** If you customize this, be sure to include something like the
     `-e` flag, otherwise individual steps failing won't fail the provisioner.
 
+-  `onlyon` (array of strings) - This is an array of 
```

Comment:
As a follow-on to the above comment, let's change this to `only_on`

Reply:
Also, your change will also affect the shell-local post-processor, so we should update those docs as well.